package-lock.json

```diff
@@ -1,39 +1,8757 @@
 {
     "name": "enketo-core",
-    "version": "5.18.1",
-    "lockfileVersion": 1,
+    "version": "6.0.0",
+    "lockfileVersion": 2,
```

Comment:
@eyelidlessness Can you sanity check that this is fine?

Reply:
I don't think this is right. It suggests the lockfile was generated with NPM 7, which will cause installs to fail. We saw this previously with libxmljs and libxslt, but I see it is also failing with the git reference to my mergexml fork as well.

Are you using Volta? If not, I'd highly recommend it as it will (quickly, unlike nvm) use the Node/NPM versions pinned in package.json.